src/Data/List/Properties.agda

```diff
@@ -681,6 +681,9 @@ concat-[-] : concat {A = A} ∘ map [_] ≗ id
 concat-[-] [] = refl
 concat-[-] (x ∷ xs) = cong (x ∷_) (concat-[-] xs)
 
+concat-[_] : concat {A = A} ∘ [_] ≗ id
```

Comment:
I would really like to use the same naming convention as `concat-[-]` above, which really should be called `concat-map-[-]`. However, I don't see a way to do so in the short-term. My suggestions would therefore be keep the name `concat-[_]` and deprecate the name `concat-[-]` to either

1. `concat-map-[_]` to be consistent with the new name
2. `concat-map-[-]` with the aim of eventually deprecating the new name `concat-[_]` to concat-[-]`

Reply:
If we want to start that process in this pull request I prefer the first option

Reply:
One way to move in favour of 2 would be to label this v3.0, and simply make the `breaking` change...
... but it occurs to me that neither option actually precludes an eventual deprecation in favour of the `[-]` idiom?

I'll go for 1.